### ISSUE TITLE
[Feature Request] Generate Views Angular 2

### ISSUE THREAD

**Issue by jmurphzyo** [NSwag/NSwag#523]
I don't know how hard the implementation would be or if it is currently possible but being able to generate views for the client templates would be awesome.

**Comment by rsuter** [comment]
How exactly would that look like? Generate CRUD view models? Can you provide a sample?

**Comment by jmurphzyo** [comment]
Yeah exactly and the view data would be based upon the DTO's that are already being generated. It does bring in more complexities with having to deal with multiple files which could be an ICollection of strings.
